src/python/platforms/fuchsia/device.py

```diff
@@ -24,6 +24,9 @@
 from google_cloud_utils import gsutil
```

Comment:
If this file gets imported (directly or indirectly) by appengine code there will be issues.
Maybe you can stop them by doing something like [this](https://github.com/google/clusterfuzz/pull/471/files#diff-8703e56d42b3f1a98a675c86907e8b90L24)

Reply:
ie: import it in code that won't get run on appengine.